internal/pkg/service/buffer/api/service/mapper/receiver_test.go

```diff
@@ -9,16 +9,16 @@ import (
 	"github.com/keboola/keboola-as-code/internal/pkg/idgenerator"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/api/gen/buffer"
 	. "github.com/keboola/keboola-as-code/internal/pkg/service/buffer/api/service/mapper"
+	bufferDependencies "github.com/keboola/keboola-as-code/internal/pkg/service/buffer/dependencies"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/key"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/model"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/model/column"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/common/dependencies"
 )
 
 func TestReceiverModel(t *testing.T) {
 	t.Parallel()
 
-	d := dependencies.NewMockedDeps(t)
+	d := bufferDependencies.NewMockedDeps(t)
```

Comment:
Potrebujem pridat dalsie mocked dependencies pre buffer service.
Doteraz boli vsetky mocked deps v jednom pkg, aj ked produkcne deps su podelene.
Tak som aspon oddelil mocked deps pre buffer, nech sa to nezhorsuje.
Tj. v tomto PR je len presunutie mocked deps, ktore su len pre buffer service.